src/services/system/hooks/index.js

```diff
@@ -66,7 +66,7 @@ exports.before = {
 };
 
 exports.after = {
-	all: [iff(isProvider('external'), [discard('ldapConfig.searchUserPassword')])],
```

Comment:
I like the approach.
`oauthConfig.clientId` is encrypted in the DB as well, like it is a secret. I'm not sure, if that is necessary, but I guess, better be safe than sorry. So I'd remove that from the result as well.

Currently `config.clientSecret` and `config.clientId` can exist as well. That is supposed to be harmonized soon, but so far we should remove those as well.

Reply:
I mentioned two more. Sorry, for the missunderstanding. Currently there is a slightly different configuration for systems that are handled by keycloak and those handled by dBildungscloud server (keycloak is not in prod yet, so it is less severe)

```
discard('config.clientId'),
discard('config.clientSecret'),
```

Reply:
sure thing! and sorry, I missed that.